app/browser/api/ledger.js

```diff
@@ -995,7 +995,7 @@ const pageDataChanged = (state, viewData = {}, keepInfo = false) => {
 }
 
 const backupKeys = (state, backupAction) => {
-  const date = moment().format('L')
+  const date = format(new Date(), 'MM/DD/YYYY')
```

Comment:
Most of the world uses `DD/MM/YYYY` format... does `moment().format('L')` always return the date in `MM/DD/YYYY` format? or does it respect the user's locale

Reply:
yes, the 'L' parameter forces month numeral/day of month/year. 

ref: https://momentjs.com/docs/#/manipulating/local/ and search for "Localized formats".